base_continent/models/country.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# © 2014-2016 Camptocamp SA (Author: Romain Deheele)
+# © 2017 senseFly, Amaris (Author: Quentin Theuret)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+from odoo import fields
```

Comment:
Please check the import order https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#imports